flask_praetorian/decorators.py

```diff
@@ -86,7 +87,8 @@ def wrapper(*args, **kwargs):
                 not current_guard().roles_disabled,
                 "This feature is not available because roles are disabled",
             )
-            role_set = set([str(n.value) if isinstance(n, Enum) else str(n) for n in required_rolenames])
+            role_set = set([str(n.value) if isinstance(n, Enum)
```

Comment:
I'm not sure why this line and others in this module were altered with a different style

Reply:
The idea behind it was that the list comprehension is more readable when the else and the if statement are split into two lines. I can turn it into a one-liner again if you want.

Reply:
When breaking it up into multiple lines to improve readability, I prefer:
```python
 role_set = set([
    str(n.value) if isinstance(n, Enum)
    else str(n)
    for n in required_rolenames
])
```
However, for these I think the readability on one line is fine